usr/share/rear/prep/BORG/default/100_set_vars.sh

```diff
@@ -13,8 +13,13 @@ borg_set_vars
 # combined string of "BORGBACKUP_USERNAME@BORGBACKUP_HOST".
 # borg_dst_dev directory will be created in later stage
 # (if not already present) by 250_mount_usb.sh script.
+
 if [[ -n $BORGBACKUP_HOST ]]; then
-    borg_dst_dev=$BORGBACKUP_USERNAME@$BORGBACKUP_HOST:
-else
-    borg_dst_dev=$BUILD_DIR/borg_backup
+   borg_dst_dev=$BORGBACKUP_USERNAME@$BORGBACKUP_HOST:
+   # We have to test if anoher port is used.
+      if [[ -n $BORGBACKUP_PORT ]]; then
+        borg_dst_dev=ssh://$BORGBACKUP_USERNAME@$BORGBACKUP_HOST:$BORGBACKUP_PORT/
```

Comment:
Is the trailing '/' really necessary ?
Normally repository path should contain full Borg repository path.
Heaving trailing '/' will make it impossible to use relative paths.

Reply:
you need this trailing "/" as it's part of the borg syntax, even with relative path,  It should be set in the  __BORGBACKUP_REPO__ otherwise you'll have this error :
with __BORGBACKUP_REPO__="~/backup"
```
borg create: error: argument ARCHIVE: Invalid location format: "ssh://rear_test@123.345.56.789:222~/backup::RearUxdc01_2"
```
or with __BORGBACKUP_REPO__ ="backup"
```
borg init: error: argument REPOSITORY: Invalid location format: "ssh://rear_test@123.345.56.789:222backup"
```
I think a remark on the importance on the first slash in the comment above the __BORGBACKUP_REPO__ could be a good thing. Don't you think so ?
About borg and remote repositories:
https://borgbackup.readthedocs.io/en/stable/quickstart.html#remote-repositories

__Remote repositories with relative paths, alternative syntax with port__:
```
ssh://user@host:port/./path/to/repo - path relative to current directory
ssh://user@host:port/~/path/to/repo - path relative to user’s home directory
ssh://user@host:port/~other/path/to/repo - path relative to other’s home directory
```

Reply:
@patlegu 
FYI in general regarding explanatory comments see
https://github.com/rear/rear/wiki/Coding-Style
i.e. explanatory comments are very much appreciated.

Reply:
@patlegu 
ok, then I'm fine with this one ;-)
Thanks for pointing this out!

And yes, comments on topic like this one are more then welcome! Please add your rationale directly to the code.

Thanks

V.